main/src/library/File.flix

```diff
@@ -840,4 +840,29 @@ namespace File {
                 Err(getMessage(ex))
         }
 
+    ///
+    /// Moves a file or directory from path `src` to path `dst`.
+    ///
+    /// Returns `Ok(true)` if `src` was moved, and `dst` did not already exist.
+    /// Returns `Err(msg)` if `src` was not moved, because `dst` already exists or is a subset of `src`.
+    ///
+    pub def move(src: {src :: String} , dst: String): Result[Bool, String] = ???
```

Comment:
The return types were not specified in the issue. I'm a bit unsure of what it should be, since I do not see a use for `Ok(false)` here, if the function should fail, if `dst` already exists or is a subset of `src`.

Reply:
Sounds like a use case for `Result[Unit, String` :)

Reply:
Yes, good idea :)